proto/provenance/expiration/v1/expiration.proto

```diff
@@ -0,0 +1,43 @@
+// Since: provenance 1.12
+syntax = "proto3";
+package provenance.expiration.v1;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/any.proto";
+import "google/protobuf/timestamp.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+option go_package          = "github.com/provenance-io/provenance/x/expiration/types";
+option java_package        = "io.provenance.expiration.v1";
+option java_multiple_files = true;
+
+// Params defines the set of params for the expiration module.
+message Params {
+  option (gogoproto.equal)            = true;
+  option (gogoproto.stringer)         = true;
+  option (gogoproto.goproto_stringer) = false;
+
+  // the deposit amount for holding module assets on chain
+  cosmos.base.v1beta1.Coin deposit = 1 [(gogoproto.nullable) = false];
+  // duration is the period upon which an expiration will stay on chain
+  // when the period elapses the module asset is up for extension or deletion
+  //
+  // the accepted values for a duration are n{h,d,w,y}
+  // where 1h = 60m, 1d = 24h, 1w = 7d (or 168h), 1y = 365d (or 8760h)
```

Comment:
Please use sentences in all these proto comments. Line spacing isn't maintained everywhere.